### DISCUSSION TITLE
Show integrations enabled through yaml in the UI.

### DISCUSSION THREAD

**Discussion by dbuezas** [home-assistant/frontend#20063] [Other feature requests]
Even when they won't have any options available, IMO it would still be useful to know they are there.

I recently had an issue with an old custom component and it took me months to find out it was one I added a couple of years ago via yaml.

**Comment by ildar170975** [comment]
Probably these yaml-configured integrations should be shown only if a corr. checkbox is marked - see a draft below:
![image](https://github.com/home-assistant/frontend/assets/71872483/e9a24e3b-934c-422b-ac54-2322b5eedc61)
